resources/public/api/conf/2.0/listSelfAssessmentPaymentsAndAllocationDetails.raml

```diff
@@ -0,0 +1,52 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+#  - errors.formatStartDate
+#  - errors.formatEndDate
+#  - errors.ruleEndDateBeforeStartDate
+  - errors.invalidDateRange
+  - errors.missingToDate
+  - errors.missingFromDate
+  - errors.clientOrAgentNotAuthorised
+  - errors.notFound
+  - errors.formatPaymentLot
+  - errors.formatPaymentLotItem
+displayName: "List Self Assessment Payments & Allocation Details [test only]"
+description: This endpoint enables you to list the payments and allocation details of one or more liabilities. A National Insurance number, start date and end date must be provided.
+(annotations.sandboxData): !include scenarios/default.md
+(annotations.scope): "read:self-assessment"
+securedBy: [ sec.oauth_2_0: { scopes: [ "read:self-assessment" ] } ]
+queryParameters:
+  from:
```

Comment:
Appreciate the spec has been shifting under our feet, but these parameters should be `dateFrom` and `dateTo`.